core/scripts/chaincli/handler/keeper.go

```diff
@@ -148,8 +151,40 @@ func (k *Keeper) approveFunds(ctx context.Context, registryAddr common.Address)
 	log.Println(registryAddr.Hex(), ": KeeperRegistry approved - ", helpers.ExplorerLink(k.cfg.ChainID, approveRegistryTx.Hash()))
 }
 
-// deployRegistry2 deploys a version 1.2 keeper registry
-func (k *Keeper) deployRegistry2(ctx context.Context) (common.Address, *registry12.KeeperRegistry) {
+// deployRegistry20 deploys a version 2.0 keeper registry
+func (k *Keeper) deployRegistry20(ctx context.Context) (common.Address, *registry20.KeeperRegistry) {
+	registryLogic := k.cfg.KeeperRegistryLogicAddr
+	if registryLogic == "" {
```

Comment:
don't think we need this as a config argument. If someone has logic deployed, they should also have main registry deployed and can pass that address directly